### PR TITLE
docs: update

### PR DESCRIPTION
**Pull request by wfxr**
Signed-off-by: Wenxuan Zhang <wenxuangm@gmail.com>

## Check list

- [x] I have read through the [README](https://github.com/hackathon-2022-ticli/ticli/blob/master/README.md) (especially F.A.Q section)
- [x] I have searched through the existing issues or pull requests
- [x] I have performed a self-review of my code and commented hard-to-understand areas
- [x] I have made corresponding changes to the documentation (when necessary)

## Type of change

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Breaking change
- [x] Documentation change
- [ ] CICD related improvement

## Test environment

- OS
    - [ ] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Others:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,64 +18,31 @@
...
-#### On macOS
+#### On Arch Linux
 
-You can install `ticli` with Homebrew:
+`ticli` is available in the Arch User Repository. To install it from [AUR](https://aur.archlinux.org/packages/ticli-git):
 
 ```
-brew tap hackathon-2022-ticli/homebrew-ticli
-brew install ticli
+paru -S ticli-git
```
Comment (Bowser1704) [inline README.md:45]: add `yay -S ticli`

File: README.md [hunk 2]
```diff
@@ -18,64 +18,31 @@
...
-paru -S ticli-git
+brew tap hackathon-2022-ticli/homebrew-ticli
+brew install ticli
 ```
 
-#### On macOS
+#### On Arch Linux
 
-You can install `ticli` with Homebrew:
+`ticli` is available in the Arch User Repository. To install it from [AUR](https://aur.archlinux.org/packages/ticli-git):
```
Comment (Bowser1704) [inline README.md:42]: Maybe using `ticli` will be better.

File: README.md (hunk 1 above)
Comment (wfxr) [reply README.md]: added

File: README.md (hunk 2 above)
Comment (wfxr) [reply README.md]: updated

**Pull request merged**